src/olympia/activity/tests/test_models.py

```diff
@@ -230,6 +231,35 @@ def test_user_log_as_argument(self):
         entries = ActivityLog.objects.for_user(user)
         assert len(entries) == 1
 
+    def test_ip_log(self):
+        addon = Addon.objects.get()
+        assert IPLog.objects.count() == 0
+        # Creating an activity log for an action without store_ip=True doesn't
+        # create an IPLog.
+        assert not getattr(amo.LOG.REJECT_VERSION, 'store_ip', False)
+        with core.override_remote_addr('127.0.4.8'):
+            activity = ActivityLog.create(
+                amo.LOG.REJECT_VERSION,
+                addon,
+                addon.current_version,
+                user=self.request.user,
+            )
+        assert IPLog.objects.count() == 0
+        # Creating an activity log for an action *with* store_ip=True *does*
+        # create an IPLog.
+        assert getattr(amo.LOG.ADD_VERSION, 'store_ip', False)
+        with core.override_remote_addr('15.16.23.42'):
```

Comment:
Nit: You could also use a variable for this IP address for the same reason.

Reply:
76a5f405685625eb007c9521981f1d7483755b2c